web_poet/serialization/api.py

```diff
@@ -129,18 +129,18 @@ def serialize(deps: List[Any]) -> SerializedData:
     return result
 
 
-def _load_type(type_name: str) -> type:
+def load_type(type_name: str) -> type:
```

Comment:
What do you think about renaming it to load_class? See e.g. https://docs.python.org/3/library/types.html module - it is called "types", and there are CoroutineType, etc., but the functions use "class".

Reply:
It seems "types" are used more in context of some definitions, which then used in isinstance/issubclass/typing checks. It seems when you talk about objects (e.g. class objects), "class" is more commonly used.